app/helpers/script_helper.rb

```diff
@@ -16,12 +16,19 @@ def render_javascript_pack_once_tags(...)
       concat javascript_assets_tag
       @scripts.each do |name, (url_params, attributes)|
         asset_sources.get_sources(name).each do |source|
+          crossorigin = true if local_crossorigin_sources?
+          integrity = asset_sources.get_integrity(source)
+
+          if attributes[:preload_links_header] != false
```

Comment:
this means that `preload_links_header: nil` will add the preload header, that seems counterintuitive to me?

Reply:
> this means that `preload_links_header: nil` will add the preload header, that seems counterintuitive to me?

That would be the same behavior as the default `javascript_include_tag` behavior [[1]](https://github.com/rails/rails/blob/9a9222874ade2d6f300647897a8ad99e1f05aa32/actionview/lib/action_view/helpers/asset_tag_helper.rb#L117) [[2]](https://github.com/rails/rails/blob/main/railties/lib/rails/application/configuration.rb#L201), though technically it's configurable and respects the configured default.

Reply:
shoudl we use the same `presence` trick?
```suggestion
          if attributes[:preload_links_header].present?
```

Reply:
I think that'd be the opposite behavior from what we expect, since we want `:preload_links_header` to be opt-out, not opt-in.